backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -139,6 +124,7 @@ export class AdminResolver {
     return pendingCreationsPromise.reverse()
   }
 
+  @Authorized([RIGHTS.SEARCH_USERS])
```

Comment:
This seems to be the wrong right - every function gets its own right.

Reply:
OK, I was not sure while implementing it, I understood that SEARCH_USERS is the actual Admin role / right